lib/rust/parser/src/syntax/tree/visitor/Cargo.toml

```diff
@@ -13,9 +13,9 @@ license-file = "../../LICENSE"
 proc-macro = true
 
 [dependencies]
-proc-macro2 = "1.0"
+proc-macro2 = { workspace = true }
```

Comment:
Why is there a need to change `lib/rust/parser` when the change is happening somewhere completely else?

Reply:
Not really a need, it is just w part of incremental refactoring of how we manage our dependencies. Unless there is reason not to do so, dependencies are configured at the workspace level and shared across all the crates.
In some cases this allows to reduce recompilation (e.g. when compiling different targets could cause dependencies to be included with different configurations, triggering long chains of rebuilds).